src/containers/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+export default class ErrorBoundary extends React.Component {
+    state: { hasError: Boolean };
+    constructor(props: React.PropsWithChildren<{}>) {
+        super(props);
+        this.state = { hasError: false };
+    }
```

Comment:
лайфхак, ти можеш не писати конструктор якщо в тебе TypeScript:
```suggestion
    state = { hasError: false }
```